kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSession.scala

```diff
@@ -49,6 +49,7 @@ abstract class KyuubiSession(
       f
     } catch {
       case t: Throwable =>
+        error(s"Handling $user session[$handle] exception:", t)
```

Comment:
for exceptions that will be re-thrown, seems not need to print stacktrace here

Reply:
thanks, I see